samples/template/README.md

```diff
@@ -3,3 +3,102 @@
 Start your own project with this template.
 
 See [documentation](https://insight-platform.github.io/Savant/) for more information.
+
+## Development Quick Start
+
+Instructions below assume current current platform is x86 and current directory is `template`.
+
+In case the `template` sample was copied into custom directory, `devcontainer.json` config will need to be updated. E.g. directory name is `my-module`, then
+
+1. Update `--network` value in `runArgs`
+
+```
+"runArgs": [ "--gpus=all", "--network=my-module_network" ],
+```
+
+2. Update zmq sockets volume source in `mounts`
+
+```
+{
+    "source": "my-module_zmq_sockets",
+    "target": "/tmp/zmq-sockets",
+    "type": "volume"
+},
+```
+
+Subsections below describe alternative ways to realize the general development workflow:
+
+1. Start jaeger and module processes
+2. The client sends input image to the module and receives module results (image, metadata, logs)
+3. Changes in pyfuncs, drawfunc, pre- and postprocessing code are loaded at runtime when module receives input from the client
+4. Changes in module configuration require module container restart
+5. Changes in module image, e.g. changes in `requirements.txt`, require module image rebuild
+
+### Download sample video
+
+URI-Input script demonstration requires a sample video.
+
+```
+curl -o assets/test_data/elon_musk_perf.mp4 https://eu-central-1.linodeobjects.com/savant-data/demo/elon_musk_perf.mp4
+```
+
+### Docker Compose
+
+1. Start Jaeger container
```

Comment:
What is the reason to start jaeger?

Reply:
Module that has telemetry configured panics in case the endpoint is inaccessible.

Reply:
Better to explain explicitly: `required by a module configured with jager telemetry`.